src/StateService/StatePlugin.cs

```diff
@@ -22,29 +23,59 @@ public class StatePlugin : Plugin, IPersistencePlugin
         public override string Name => "StateService";
         public override string Description => "Enables MPT for the node";
 
-        private IActorRef store;
+        internal IActorRef Store;
+        internal IActorRef Verifier;
 
         protected override void Configure()
         {
             Settings.Load(GetConfiguration());
+            RpcServerPlugin.RegisterMethods(this);
         }
 
         protected override void OnPluginsLoaded()
         {
-            store = System.ActorSystem.ActorOf(StateStore.Props(System, Settings.Default.Path));
+            Store = System.ActorSystem.ActorOf(StateStore.Props(System, this, Settings.Default.Path));
         }
 
         public override void Dispose()
         {
             base.Dispose();
-            System.EnsureStoped(store);
+            System.EnsureStoped(Store);
+            if (Verifier != null) System.EnsureStoped(Verifier);
         }
 
         void IPersistencePlugin.OnPersist(Block block, DataCache snapshot, IReadOnlyList<ApplicationExecuted> applicationExecutedList)
         {
             StateStore.Singleton.UpdateLocalStateRoot(block.Index, snapshot.GetChangeSet().Where(p => p.State != TrackState.None).ToList());
```

Comment:
@shargon If we call in this way. It will affect the performance of block persisting.